README.rst

```diff
@@ -207,50 +207,52 @@ Crafted with ♥ in Berlin.
 
 .. |Brand| replace:: *fireant*
 
-.. |FeatureDataSet| replace:: *Data Set*
+.. |FeatureDataSet| replace:: *DataSet*
```

Comment:
can you please revert the changes on this file and manually add the new classes? These were adjusted from the generated values to represent the place where you can import these classes when using fireant as a library, rather than the actual package location. This reflects the setup.py configuration, which you also need to make sure includes your new files.

Some of the classes are imported to an easier to reach package so users of the fireant library don't have to go digging deep into packages and can streamline their imports from the fireant package by importing from a single package like `from fireant import {0}, {1}, ...`

Reply:
Okay makes sense. I just thought it would be nice if you could actually click the API references and then be forwarded to their documentation.

I will also edit the setup.py